lighthouse-extension/app/src/lighthouse-ext-background.js

```diff
@@ -73,19 +73,17 @@ async function runLighthouseInExtension(options, categoryIDs) {
  * Run lighthouse for connection and provide similar results as in CLI.
  * @param {Connection} connection
  * @param {string} url
- * @param {{flags: LH.Flags} & {outputFormat: string, logAssets: boolean}} options Lighthouse options.
+ * @param {LH.Flags & {logAssets: boolean}} flags Lighthouse flags plus logAssets
```

Comment:
`logAssets`  could also be a fifth parameter to the function, or we could wait for some future refactor of `saveArtifacts`/`saveAssets`/`logAssets` (or this could `@return {Promise<LH.RunnerResult|void>}` and the caller could pull the `artifacts` off there)